gprofiler/metadata/system_metadata.py

```diff
@@ -194,15 +205,39 @@ class SystemInfo:
     spawn_uptime_ms: float
 
 
+def get_windows_network_details() -> Tuple[str, str]:
```

Comment:
Definition can be placed under `if is_windows()`. If needed only on Windows - strive to place under `if is_windows()`